src/GrassPokemon.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class GrassPokemon extends Pokemon {
+    GrassPokemon(String name, Integer hp, Integer xp, String food, String sound) {
+        super(name, hp, xp, food, sound);
+    }
+
+    public void leafStorm(Pokemon name, Pokemon enemy){
+        System.out.println(name.name + " attacks " + enemy.name + " with leafstorm");
+
+        switch (enemy.getType()) {
```

Comment:
Je hebt de logica van de aanvallen goed toegepast, maar het is wel veel kopieren en plakken. 
Het handiger om de herhalende code dan naar een helper methode te verplaatsen en die op alle plekken aan te roepen. Dan heb je nog maar 1 regeltje "herhaling" ipv een complete switch statment